originals/d.yaml

```diff
@@ -507,6 +507,16 @@
     themes:
     - Fantasy
 
+- name: Dragon Alpha
```

Comment:
external is required - needs wikipedia or website

Reply:
Well, as far as a website is concerned, there's the GitHub repository for the game (https://github.com/jaredplumb/Dragon-Alpha), but that's the closest I have... I don't think its old page on the developer's website (Dracosoft) has been archived so I can't give a Wayback Machine link, and other than that there are just third-party websites that have the game uploaded or have information about it.

Reply:
I think a third party website is preferred. The repo does give good info though